### PR TITLE
Update sql-agent-extension-automatic-registration-all-vms.md

### PR DESCRIPTION
**Pull request by moematsuo**
Added a note to clarify when the automatic registration is executed.

### CODE REVIEW

**Comment by Court72** [comment]
@adbadram

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/sql-agent-extension-automatic-registration-all-vms.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ When automatic registration is enabled, a job runs monthly to detect whether or
 
 Once automatic registration is enabled for a subscription, all current and future VMs that have SQL Server installed will be registered with the SQL IaaS Agent extension **in lightweight mode without downtime, and without restarting the SQL Server service**. You still need to [manually upgrade to full manageability mode](sql-agent-extension-manually-register-single-vm.md#upgrade-to-full) to take advantage of the full feature set. The license type automatically defaults to that of the VM image. If you use a pay-as-you-go image for your VM, then your license type will be `PAYG`, otherwise your license type will be `AHUB` by default. 
 
+> [!NOTE]
+> There might be scenarios where user would have deployed a SQL VM which has IaaS extension preinstalled. Sometimes its in a failure state or sometimes they will remove it. Automatic registration will check the registry to see if SQL Server is installed on a VM and if extension is not there then it would register it in lightweight mode.
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/sql-agent-extension-automatic-registration-all-vms.md]:
```suggestion
> If you deployed a SQL Server VM with the IaaS extension preinstalled, and the extension is in a failure state or it was removed, automatic registration will check the registry to see if SQL Server is installed on a VM, and register it in lightweight mode.
```

Comment (bluefooted) [reply azure-sql/virtual-machines/windows/sql-agent-extension-automatic-registration-all-vms.md]:
```suggestion
> If you deployed a SQL Server VM with a marketplace image which has the SQL IaaS agent extension preinstalled, and the extension is in a failure state or it was removed, automatic registration will check the registry to see if SQL Server is installed on the VM and register it in lightweight mode.
```

**Comment by MashaMSFT** [comment]
Hi @moematsuo , thanks for your submission! I pulled your change into the private repo. You'll still see your changes, and be marked as a contributor, but I'll close out this PR. Please feel free to comment should you need anything further, and thanks again for improving our docs!

Hope you have a wonderful rest of your day :)
Masha from the SQL Docs team

**Pull request closed without merging**